src/ManageCourses.Api/Controllers/AccessRequestController.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ManageCourses.Api.Data;
+using GovUk.Education.ManageCourses.Domain.DatabaseAccess;
+using GovUk.Education.ManageCourses.Domain.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace GovUk.Education.ManageCourses.Api.Controllers
+{
+    
+    
+    [Route("api/[controller]")]
+    public class AccessRequestController : Controller
+    {
+        private readonly IManageCoursesDbContext _context;
+        private readonly IEmailService _emailService;
+
+        public AccessRequestController(IManageCoursesDbContext context, IEmailService emailService)
+        {
+            _context = context;
+            _emailService = emailService;
+        }
+
+        //[Authorize]
+        [HttpPost]
+        public async Task<StatusCodeResult> Index([FromForm] Api.Model.AccessRequest request) 
+        {
+            var requesterEmail = "daniel@eigencode.io"; //this.User.Identity.Name;
+            using (var transaction = ((DbContext)_context).Database.BeginTransaction()) 
+            {
+                try
+                {
+                    var requester = _context.McUsers
+                        .Include(x=>x.McOrganisationUsers)
+                        .ThenInclude(x=>x.McOrganisation).Single(x => x.Email == requesterEmail);
```

Comment:
Are we not going to use lazy loading?
I was halfway through to setting up entity framework to use that....

Reply:
Don't need lazy loading here - all elements in scope need to be retrieved. The filter (email == email) will be evaluated server-side, thanks to EF magic